src/plugins/Dashboard/index.js

```diff
@@ -175,14 +176,14 @@ module.exports = class DashboardUI extends Plugin {
   }
 
   initEvents () {
-    // const dashboardEl = this.target.querySelector(`${this.opts.target} .UppyDashboard`)
-
     // Modal open button
     const showModalTrigger = findDOMElement(this.opts.trigger)
     if (!this.opts.inline && showModalTrigger) {
       showModalTrigger.addEventListener('click', this.showModal)
-    } else {
-      this.core.log('Modal trigger wasn’t found')
+    }
+
+    if (!this.opts.inline && !showModalTrigger) {
+      console.error('Dashboard modal trigger not found, you won’t be able to select files. Make sure `trigger` is set correctly in Dashboard options')
```

Comment:
I used that initially, but then thought that there are errors that should log in production too, right? Not sure if this is the one, but.